pyproject.toml

```diff
@@ -24,8 +24,8 @@ classifiers = [
 ]
 urls = {repository = "https://github.com/patrick-kidger/quax" }
 dependencies = [
-    "jax>=0.4.38",
-    "jaxtyping>=0.2.20",
+    "jax>=0.4.38,<0.5.3",
+    "jaxtyping>=0.2.20,<0.3",
```

Comment:
I'd quite like to avoid doing a version bump (with a new release) *and* restricting the JAX and jaxtyping versions at the same time.

IIUC you want the version constraints so that CI will pass? If so then let's:
- merge this without the new release
- then update to lift these version constraints
- then do a new release

Does that sound reasonable?

Reply:
Good for step 1.